recipes/pybigwig/meta.yaml

```diff
@@ -19,7 +19,7 @@ requirements:
     - curl
     - libcurl
     - zlib
-    - numpy
+    - {{ pin_compatible('numpy') }}
```

Comment:
I don't think this is correct. Any reason why you have changed this? 
xref: https://conda-forge.org/docs/meta.html#building-against-numpy

Reply:
Click over to the circleci error from the previous commit. For whatever reason it appears to have been building against one version (1.9.3 for some reason) and then testing/running against another (1.15.0). That won't work in C unless things are ABI compatible. This change seemed odd, but it got everything to the same version.

Reply:
https://circleci.com/gh/bioconda/bioconda-recipes/23223?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link